getting-started/README.md

```diff
@@ -5,16 +5,18 @@
 To get started with OPEA you need the right hardware and basic software setup.
 
 Hardware Requirements: For the hardware configuration, If you need Hardware Access visit the Intel Tiber Developer Cloud to select from options such as Xeon or Gaudi processors that meet the necessary specifications.
-Software Requirements: Please refer to the Support Matrix[ Hyper link needed]  to ensure you have the required software components in place.
+
+Software Requirements: Refer to the Support Matrix [Hyper link needed]  to ensure you have the required software components in place.

```

Comment:
@preethivenkatesh @tomlenth Any idea where this software support matrix might be?

Reply:
@dbkinder i have the link, do you want me to create a pr or can you please add it. https://github.com/opea-project/GenAIExamples?tab=readme-ov-file#getting-started

Reply:
Thanks. I'll add that to my PR.